factory/config.go

```diff
@@ -23,15 +23,19 @@ type Cluster struct {
 	Status           string
 	ServerIPs        []string
 	AgentIPs         []string
+	WinAgentIPs		 []string
 	NumServers       int
 	NumAgents        int
+	NumWinAgents	 int
 	RenderedTemplate string
 	ExternalDb       string
 	ClusterType      string
+	ProductType      string
+	ArchType		 string
 }
 
 func loadConfig() (*config.ProductConfig, error) {
-	cfg, err := config.LoadConfigEnv("./config")
+	cfg, err := config.LoadConfigEnv(shared.BasePath() + "/distros-test-framework/config")
```

Comment:
why doing this ?

Reply:
Updated thanks, I was trying out `.env` approach instead of config but it was not working...